app/views/map/index.html.erb

```diff
@@ -8,6 +8,8 @@
 	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.76.0/dist/L.Control.Locate.min.css" />
   	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.1/leaflet.css" />
 	
+	
+	<%= content_tag :div, '', id: 'container', data: { source: @PoI } %>
```

Comment:
'Cotainer' is basically just what a div is, I think that name is a little too unspecific here, especially since IDs need to be unique and the term is widely used in web apps. A Maybe 'poi-data' or something similar? (just you don't forget: this would need to be changed in map_build l. 85  as well)